docs/faq/README.md

```diff
@@ -57,10 +57,14 @@ class DroidList extends React.Component {
         {droids.map((droid, index, arr) =>
           <li key={index} style={{
             borderColor: 'black',
-            borderRadius: index === 0 ? '12px 12px 0 0' :
-              index === (arr.length - 1) ? '0 0 12px 12px' : '',
-            borderStyle: 'solid',
+            borderRadius: 
```

Comment:
nested ternary is ugly to begin with :) this looks better, but perhaps indent the second ternary again